CONTRIBUTING.md

```diff
@@ -31,6 +31,11 @@ here: https://guides.rubygems.org/contributing/
 
     rake setup
 
+If this fails with permission related errors, consider setting `GEM_HOME`
+environment variable and then try again:
+
+    export GEM_HOME=/tmp/rubygems.gems
+
```

Comment:
This will be fixed by #5327, but I'm happy to provide workarounds in the mean time.

But I'd rather recommend using a "per-user" Ruby installation like those provided by RMV, Rbenv, or Chruby. We could say something like

> NOTE: If this fails with permission related errors, you're most likely using a global Ruby installation (like the one packaged by your OS), which sets `GEM_HOME` to a location regular users can't write to. Consider using a Ruby version manager like RVM, Rbenv or Chruby (which install Ruby to a location regular users can write to). Alternatively, consider setting `GEM_HOME` environment variable to a writable location with something like `export GEM_HOME=/tmp/rubygems.gems` and try again.